### DISCUSSION TITLE
Can others use my email alias?

### DISCUSSION THREAD

**Discussion by veerendra2** [simple-login/app#1043] [Q&A] [answered]
Hello,
I just found your email relay app, I have 2 questions

1. Can other users use my email alias after I disable/delete that alias or while I using that same email alias? 
2. While generating email alias, does the app checks the selected random email alias already exist?

If these questions already answered in any FAQ, please refer it

Thanks,
Veerendra

**Comment by notDavid** [comment, marked as answer]
Fyi, I was looking for this also, and found this: 

> We recommend against deleting an alias as this operation isn't reversible, meaning a deleted alias can't be restored. In fact, to avoid a deleted alias from being reused by another user, a deleted alias is moved to a global "trash" and SimpleLogin make sure no one can reuse a deleted alias.
> 
> Exception is made for aliases created with your domain or subdomain: in this case, you can recreate an alias. Please note that this isn't the same as restoring an alias as all records like contacts, settings, etc associated with an alias are deleted when the alias is deleted.
> 
> Instead of deleting an alias, we suggest either disable the alias or block some of its senders.
> 
> If you really want to delete an alias, you can click on "More" button, then "Delete" button.

https://simplelogin.io/docs/getting-started/delete-alias/
